Stratis.Bitcoin.Tests/Properties/AssemblyInfo.cs

```diff
@@ -33,4 +34,7 @@
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
 [assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
\ No newline at end of file
+[assembly: AssemblyFileVersion("1.0.0.0")]
+
+// disable parallel running of tests because of concurrency issues with the log class being static (see LogsTestBase).
+[assembly: CollectionBehavior(DisableTestParallelization = true, MaxParallelThreads = 1)]
```

Comment:
Ok this is a good intensive to instance the logging freamwork
Maybe add a todo to remember to enable this back? (next commit)